streaming/base/storage/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Base module for download/upload files from/to a cloud storage."""
```

Comment:
hmm

"""Base module for downloading/uploading files from/to cloud storage."""

"""Base module for file download/upload from/to cloud storage."""

Reply:
Thanks! Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.